client/src/GlobalStyles.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+import Poppins from "./assets/poppins-regular.ttf";
+function GlobalStyles() {
+  return (
+    <Global
+      styles={css`
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+        }
+
+        :root {
+          --bg-color-main: #f2f2f2;
+          --bg-color-light: #fff;
+          --bg-color-highlight: #f27649;
+          --bg-color-dark-highlight: #f26a4b;
+          --font-color-main: #262526;
+          --font-color-menu: #fff;
+          --font-title-main: #4b4b5c;
+        }
+        @font-face {
+          font-family: "Poppins";
+          src: url(${Poppins}) format("truetype");
+        }
+
+        #root {
+          background-color: var(---bg-color-main);
+        }
+
+        body {
+          height: 100vh;
+          width: 100vw;
+          margin: 0;
+          padding: 0;
+          overflow: hidden;
```

Comment:
`height`, `width` and `overflow`: I'd put these on the watchlist of potential issues for later. Whenever there is weird scrollbars, fiddle around with these :)
Also don't expect `height: 100vh` to work perfectly on real mobile devices (not the dev toosl setting)